sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -679,7 +679,7 @@ def get_media_val_str(num_logical_ports, lane_dict, logical_idx):
     media_val_str = ''
     if (num_logical_ports > 1) and \
        (num_lanes_on_port >= num_logical_ports):
-        num_lanes_per_logical_port = num_lanes_on_port/num_logical_ports
+        num_lanes_per_logical_port = int(num_lanes_on_port/num_logical_ports)
```

Comment:
Suggest using floor division instead:

```python
num_lanes_per_logical_port = num_lanes_on_port // num_logical_ports
```

This should produce an int in both Python 2 and 3.